Taxonomy subtree deletion and clean-up (#2271)
ibexa/documentation-developer: docs/update_and_migration/from_4.2/update_from_4.2.md

If you have no access to [[= product_name =]]'s `ibexa/installer` package, database upgrade is not necessary.

### Clean-up taxonomy database

> Run the following command for each of your taxonomy,
> to be sure you don't have [Content items orphaned during deletions of subtrees](taxonomy.md#remove-orphaned-content-items):
> `php bin/console ibexa:taxonomy:remove-orphaned-content <taxonomy> --force`

Reviewer: ```suggestion
Run the following command for each of your taxonomies to ensure that there are no [Content items orphaned during deletion of subtrees](taxonomy.md#remove-orphaned-content-items):

`php bin/console ibexa:taxonomy:remove-orphaned-content <taxonomy> --force`
```
